manim/mobject/geometry.py

```diff
@@ -863,6 +864,225 @@ def __init__(self, n=6, **kwargs):
         Polygon.__init__(self, *vertices, **kwargs)
 
 
+class ArcPolygon(VMobject):
+    """A generalized polygon allowing for points to be connected with arcs.
+
+    This version tries to stick close to the way :class:`Polygon` is used. Points can be
+    passed to it directly which are used to generate the according arcs
+    (using :class:`ArcBetweenPoints`). An angle or radius can be passed to it to
+    use across all arcs, but to configure arcs individually an arc_config list has to be
+    passed with the syntax explained below.
+
+    ArcPolygons can be transformed properly into one another as well. Be
```

Comment:
Issue: I don't understand this sentence. "one another" requires two subjects, but this sentence only has one subject ("ArcPolygons") Or am I too tired and missing something :thinking:

Reply:
Well it's talking about the transformation of `one` arcpolygon into `another` arcpolygon. If you have a way to make this clearer, fire away.

Reply:
Ooooooh. Right. So here is my suggestion.
```suggestion
    Two instances of :class:`ArcPolygon` can be transformed properly into one another as well. Be
```

Reply:
Sure, taken.